backend/src/main/java/com/praksa/auction/model/Person.java

```diff
@@ -21,114 +30,5 @@ public class Person {
     @ManyToOne
     @JoinColumn(name = "addressId")
     private Address address;
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "cardId")
```

Comment:
Drop table Card and this column cardId from Person table.

Reply:
Please leave a message after dropping this table.

Reply:
I dropped the table both locally and on heroku